sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -526,6 +526,16 @@ object JdbcUtils extends Logging with SQLConfHelper {
       (rs: ResultSet, row: InternalRow, pos: Int) =>
         row.update(pos, rs.getBytes(pos + 1))
 
+    case _: YearMonthIntervalType =>
```

Comment:
Is there any side-effect for the other JDBC dialects?

Reply:
It's safe here as there are no mapping rules to `YearMonthIntervalType` at the schema side